descope/auth/auth.go

```diff
@@ -75,11 +75,40 @@ func (auth *authenticationService) WebAuthn() WebAuthn {
 	return auth.webAuthn
 }
 
-func (auth *authenticationService) Logout(request *http.Request, w http.ResponseWriter) error {
-	return auth.LogoutWithOptions(request, WithResponseOption(w))
+func (auth *authenticationService) DeleteCookies(request *http.Request, w http.ResponseWriter) error {
+	return auth.DeleteCookiesWithOptions(request, WithResponseOption(w))
 }
 
-func (auth *authenticationService) LogoutWithOptions(request *http.Request, options ...Option) error {
+func (auth *authenticationService) DeleteCookiesWithOptions(request *http.Request, options ...Option) error {
+	if request == nil {
+		return errors.MissingRequestError
+	}
+
+	cookies := make([]*http.Cookie, 0)
+	sessionCookie, _ := request.Cookie(SessionCookieName)
+	refreshCookie, _ := request.Cookie(RefreshCookieName)
+
+	if sessionCookie != nil {
+		sessionCookie.MaxAge = -1
```

Comment:
I don't think max age `-1` is a valid syntax, from what I know `maxAge=0` is the protocol 

https://stackoverflow.com/questions/15932957/difference-between-0-and-negative-value-for-setmaxage-for-cookie

Reply:
In go it is -1.

Reply:
![image](https://user-images.githubusercontent.com/5429033/196284301-3440a92e-a33c-4462-a339-738a16d9b7db.png)

Reply:
nice 👍